src/main/java/org/spongepowered/common/mixin/core/item/MixinItemArmorMaterial.java

```diff
@@ -58,4 +67,23 @@ public String getId() {
     public String getName() {
         return this.name;
     }
+
+    @Override
+    public Optional<ItemType> getRepairItemType() {
```

Comment:
It is a forge related method now, since the item can be constructed differently. What we should technically do is make the breaking change to use `ItemStackSnapshot`s instead of `ItemType`s. But I did come across this being unimplemented the other day and had planned on implementing it in a hot fix commit.